pkg/console/starter/starter.go

```diff
@@ -386,6 +386,14 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 			"CustomRouteSyncProgressing",
 			"DefaultRouteSyncDegraded",
 			"DefaultRouteSyncProgressing",
+
+			// Remove multicluster conditions until 4.15
+			"ThanosQuerierProxyServiceResolverSyncDegraded",
+			"ThanosQuerierProxyServiceResolverSyncProgressing",
+			"ManagedClusterSyncDegraded",
+			"ManagedClusterSyncProgressing",
+			"ManagedClusterConfigSyncDegraded",
+			"ManagedClusterConfigSyncProgressing",
```

Comment:
It looks like we can't remove these since the condition is getting set by the main sync loop:

https://github.com/openshift/console-operator/blob/release-4.13/pkg/console/operator/sync_v400.go#L79

Reply:
Right, we either need to keep those, or cherry-pick https://github.com/openshift/console-operator/pull/774 into here